src/core_plugins/kibana/public/dev_tools/hacks/disable.js

```diff
@@ -0,0 +1,16 @@
+import chrome from 'ui/chrome';
+import modules from 'ui/modules';
+
+modules.get('kibana').run(function ($rootScope, $location) {
```

Comment:
I would also prefer if the devtool app wasn't even in the navbar, but maybe there was a reason it was left.